packages/ethersjs/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@metamask/test-snap-ethersjs",
+  "version": "5.5.0",
+  "description": "MetaMask ethersjs Test Snap",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/test-snaps.git"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "license": "ISC",
+  "files": [
+    "dist/",
+    "images/icon.svg",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "mm-snap build --eval false",
```

```suggestion
    "build": "mm-snap build --eval false",
```
Does it fail without `--eval false`?

---

This is the same in *all* of the package.json files in test-snaps... should it not be?

---

I removed it and it still works